models/categories.model.js

```diff
@@ -0,0 +1,19 @@
+const db = require('../db/connection.js')
+
+
+const fetchCategories = (endPoint) => {
+    if(endPoint === 'categories'){
```

Comment:
id look at app.all("*") - to handle routes you havent created/404 ones. 

At the moment, this method will cause you headaches later down the line